index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
@Fakhara77 what is the purpose of this file?

Reply:
because main.js is an external file. i have access in index.html to see the result of main.js in browser

Reply:
now i have resolved issue.now my files access in main.js by using node.js